### PR TITLE
Creation an TTL file for data model for ontowikikul

### PR DESCRIPTION
**Pull request by bszady**
I have added new file in OWL/TTL format to register data model for wikikul

### CODE REVIEW

File: Data Model/ontowikikul.ttl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@prefix : <http://www.semanticweb.org/histo/ontologies/2024/1/untitled-ontology-39/> .
```
Comment (przemekgradzki) [inline Data Model/ontowikikul.ttl]:
```suggestion
@prefix : <https://onto.kul.pl/ontowikikul/> .
@prefix ontowikikul: <https://onto.kul.pl/ontowikikul/> .
```

File: Data Model/ontowikikul.ttl [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@base <http://www.semanticweb.org/histo/ontologies/2024/1/untitled-ontology-39/> .
```
Comment (przemekgradzki) [inline Data Model/ontowikikul.ttl]:
```suggestion
@base <https://onto.kul.pl/ontowikikul/> .
```

**Review by przemekgradzki — CHANGES_REQUESTED** [review]

**Comment by bszady** [comment]
@przemekgradzki @bernardyn I think it is ready, please accept. Further work in the next pull request

**Review by bernardyn — DISMISSED** [review]
looks fine

**Review by przemekgradzki — APPROVED** [review]
OK after applying  serialization (7aa24eb)

**Review by bernardyn — APPROVED** [review]
serialisation always :) approve

**Pull request merged**